shiki/WordPress-Android
# Preview in Giphy Picker

![dec-02-2018 07-15-15](https://user-images.githubusercontent.com/198826/49340854-8d8d1100-f602-11e8-9f7e-eaaa96af45fe.gif)

This is a part of the Giphy picker feature (wordpress-mobile#8627). It adds previewing of GIFs using the _Preview_ button or long pressing a single GIF.

- The `develop` branch from [wordpress-mobile/WordPress-Android](/wordpress-mobile/WordPress-Android/tree/develop) was merged before the changes. Merge commit is c79ba5f.
- The Preview-related changes start from commit 624d1d7.
- Preview functionality uses the existing `MediaPreviewActivity`

### Testing 

1. Create a [Giphy API key](https://developers.giphy.com/docs/) and paste it in your `gradle.properties` as the value for `wp.giphy.api_key`
2. Go to the Media Library and click on the _Add_ button at the top right. Select the _Choose from Giphy_ option.
3. In the Giphy picker, perform a search and select some items.
4. Press the _Preview_ button. The selected GIFs should be shown on a different page. 

Also, try long pressing on a single GIF.
